src/lib/pmd/PmdCatalogWrapper.ts

```diff
@@ -39,14 +40,8 @@ export class PmdCatalogWrapper extends PmdSupport {
 		return PmdCatalogWrapper.readCatalogFromFile();
 	}
 
-	private static getCatalogName(): string {
-		// We must allow for env variables to override the default catalog name. This must be recomputed in case those variables
-		// have different values in different test runs.
-		return process.env.PMD_CATALOG_NAME || PMD_CATALOG;
-	}
```

Comment:
Maybe I'm just not there yet, but don't we still need this? How will our tests be able to override the name?

Reply:
We don't need to override the PmdCatalog.json file, but the Catalog.json file instead.  I did not do that in the prior PR, just catching up now.  In LocalCatalog you will see an equivalent method.